solutions/instance/variables.tf

```diff
@@ -12,11 +12,11 @@ variable "existing_resource_group" {
 
 variable "resource_group_name" {
   type        = string
-  description = "The name of a new or an existing resource group in which Cloud Object Storage instance will be provisioned."
+  description = "The name of a new or existing resource group to provision resources in."
 }
 
 variable "resource_keys" {
-  description = "The definition of any resource keys to be generated."
+  description = "The JSON definition of the resource keys to generate. [Learn more](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/resource_key)."
```

Comment:
same comment as below about json

Reply:
I just took out `JSON`, which is what we had anyway.